### DISCUSSION TITLE
Because the payment failed, I am no longer a Pro member, and the Copilot I paid annually is no longer available.

### DISCUSSION THREAD

**Discussion by GuangminJu** [community/community#67772] [New to GitHub]
Because the payment failed, I am no longer a Pro member, and the Copilot I paid annually is no longer available.
Now I've paid for the Pro membership, but Copilot doesn't seem to be coming back.

**Comment by queenofcorgis** [comment]
Hi @GuangminJu ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
